src/app/components/shell/AppShell.vue

```diff
@@ -217,6 +313,16 @@ html, body
 		top: $shell-top-nav-height
 		left: calc(var(--shell-content-sidebar-width) + var(--shell-cbar-width))
 
+.quest-window
```

Comment:
can we make chat-window and quest-window use the same styling so that less code loads?